src/homepagecounter.js

```diff
@@ -0,0 +1,18 @@
+export function updateitemscount() {
+  const counter = document.body.querySelectorAll('.card');
+
+  const navli = document.getElementById('counter');
+  navli.innerHTML = `Spaceships ( ${counter.length} )`;
+}
+
+export function likecount() {
+  // const likebutton = document.querySelectorAll('.likebutton');
+  // const likebuttoncounter = document.querySelectorAll('.likebuttoncounter');
+
+  // likebutton.forEach((button) => {
+  //   button.addEventListener('click', () => {
+  //     const likebuttoncounter = button.nextElementSibling;
+  //     likebuttoncounter.innerHTML = 'Liked';
+  //   });
+  // });
```

Comment:
It would be nice if you do not push the commented code to the repository. Please remove this part.